ModsenOnlineStore.EmailAuthentication.Infrastructure/Services/VerificationCodeGeneratior.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using ModsenOnlineStore.EmailAuthentication.Application.Interfaces;
+
+namespace ModsenOnlineStore.EmailAuthentication.Infrastructure.Services
+{
+    public class VerificationCodeGeneratior : IVerificationCodeGeneratior
```

Comment:
put it into Application layer

Reply:
Could you please tell me why it should be stored in the Application layer?

Reply:
VerificationCodeGeneratior is the implementation interface IVerificationCodeGeneratior. And as implementation as far as I know it should be stored in the Infrastructure layer.

Reply:
Infrastructure layer should store components that operates with external systems e.g. database. This class just stores logic for generating random strings. But okay, let it be here if it is used only by the mail service.